shopinvader_product_template_multi_link/tests/test_product.py

```diff
@@ -6,12 +6,49 @@
 
 
 class ProductCase(ProductCommonCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
Maybe 

```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```

Reply:
it's not needed because it's done on the base mixin for tests :wink:

Reply:
thanks :+1: